README.md

```diff
@@ -108,7 +111,14 @@ Proc. IEEE, 102(9):1367-1388, 2014.
 - <a name="Vanarse16fnins"></a>Vanarse, A., Osseiran, A., Rassau, A,  
 *[A Review of Current Neuromorphic Approaches for Vision, Auditory, and Olfactory Sensors](http://dx.doi.org/10.3389/fnins.2016.00115)*,
 Front. Neurosci. (2016), 10:115.
-
+- <a name="Posch09Sensors"></a>
+Posch, C., Matolin, D., Wohlgenannt, R., Maier, T., Litzenberger, M., *[A Microbolometer Asynchronous Dynamic Vision Sensor for LWIR](https://doi.org/10.1109/JSEN.2009.2020658)*, IEEE Sensors Journal, 9 (2009), 6; S. 654 - 664.
+- <a name="Belbachir10icdsc"></a>Belbachir, A., Pflugfelder, R., Gmeiner, P., *[A Neuromorphic Smart Camera for Real-time 360deg distortion-free Panoramas](https://doi.org/10.1145/1865987.1866022)*,
+IEEE Conference on Distributed Smart Cameras, USA (2010).
+- <a name="Graf14CSEDU"></a>Graf, R., King, R., Belbachir, A., *[Braille Vision Using Braille Display and Bio-inspired Camera](http://dx.doi.org/10.5220/0004949302140219)*, 
+International Conference on Computer Supported Education CSEDU, Barcelona, Spain, SCITEPRESS Digital Library, (2014), S. 214 - 219.
+- <a name="Belbachir08icecs"></a>Belbachir, A., Hofstaetter, M., Reisinger, K., Litzenberger, M., Schoen, P., *[High-Precision Timestamping and Ultra High-Speed Arbitration of Transient Pixels' Events](https://doi.org/10.1109/ICECS.2008.4674996)*, 
+Proc. of the 15th IEEE International Conference on Electronics, Circuits and Systems", 2008.
```

Comment:
Are these papers best classified in this category of "Neuromorphic Systems"? Is it more appropriate to place them in a different category? This section is intended for papers on general event-based vision, rather than on specific topics such as 360 panoramic reconstruction or high-speed arbitration of transient pixels' events. Please re-organize.

Reply:
Hmm, where would you put it? Should I make new categories?